docs/docker.md

```diff
@@ -0,0 +1,169 @@
+# Prepare docker image
```

Comment:
make the steps here a bash scripts under `scrips/docker/`

Reply:
A tricky point to construct bash script under `scripts/docker` is, we should do float value comparing for DOCKER_API_VERSION.

- To sync docker API version between internal and external code, the docker API version in the code level should be 1.43.
- Some of our ARM server installed docker package with API version as 1.41. Note that it's hard to upgrade into 1.43.
- We could fix DOCKER_API_VERSION as 1.41, but we would lose capability to use docker-based CO if the machine installed docker lower than 1.41.

Reply:
Self-answer: seems `sort` function have version sort as its flag.